onnxruntime/core/mlas/lib/mlasi.h

```diff
@@ -2280,7 +2280,12 @@ MLAS_FLOAT32X4
 MlasMultiplyAddFloat32x4(MLAS_FLOAT32X4 Vector1, MLAS_FLOAT32X4 Vector2, MLAS_FLOAT32X4 Vector3)
 {
 #if defined(MLAS_NEON_INTRINSICS)
+#if defined(__ANDROID__) && defined(MLAS_TARGET_ARM)
```

Comment:
How about non-Android ARMv7 ?

Reply:
haven't tested that yet. the [docs for vfmaq_f32](https://developer.arm.com/architectures/instruction-sets/intrinsics/#q=vfmaq_f32) seem to imply that it is available on v7. I'm assuming it is an Android-specific limitation for now. we can adjust the ifdef if needed.